tests/sentry/event_manager/interfaces/snapshots/test_stacktrace/test_serialize_returns_frames.pysnap

```diff
@@ -1,13 +1,13 @@
 ---
-created: '2019-03-14T17:12:36.283191Z'
+created: '2019-12-19T00:38:25.116607Z'
 creator: sentry
 source: tests/sentry/event_manager/interfaces/test_stacktrace.py
 ---
 errors: null
-get_stacktrace: "Stacktrace (most recent call last):\n\n  File \"foo.py\", line 1"
+get_stacktrace: "Stacktrace (most recent call first):\n\n  File \"foo.py\", line 1"
```

Comment:
Where are these changes coming from ?

Reply:
It comes from a difference in event.platform, which causes stacktraces to be handled differently. Previously the event platform was incorrectly set to None, now it is "other" (which is more correct since that matches what's in the data).